package.json

```diff
@@ -21,11 +22,14 @@
   },
   "devDependencies": {
     "bluebird": "^2.9.12",
+    "chai": "^2.2.0",
```

Comment:
Actually, question for you. Is it considered best practice to not introduce new dependencies because we already established an assertion library to use? Does it make more sense in this case to use the lib already built in and if we decide to move onto something like chai, to do a full refactor then? Trying to understand why I should use `chai` as it allows for custom error messages (and I feel it's better than should.js).